pyroute2/netlink/generic/wireguard.py

```diff
@@ -189,7 +199,7 @@ def encode(self):
                         self['family'] = AF_INET
                         self['addr4'] = unpack('>I',
                                                inet_aton(self['addr']))[0]
-                        self['addr6'] = bytearray(4)  # Set 4 bytes to NULL
+                        self['addr6'] = b'\x00\x00\x00\x00\x00\x00\x00\x00'
```

Comment:
Previous code was not working?

Reply:
It worked but I think NLA auto-padding was our guardian angel.

Reply:
Tested with both IPv4 and IPv6 addresses on both Python versions. It works properly.